.pre-commit-config.yaml

```
@@ -0,0 +1,32 @@
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v1.2.3
+    hooks:
+      - id: trailing-whitespace
+      - id: check-ast
+      - id: end-of-file-fixer
+      - id: mixed-line-ending
+
+  - repo: https://github.com/PyCQA/flake8
+    rev: 3.9.2
+    hooks:
+      - id: flake8
+        entry: flake8
+        types: [file, python]
```

I think we don't need `.flake8` file, because I see no reason to ignore `E203`, so we can add here 
```suggestion
      - id: flake8
        entry: flake8
        types: [file, python]
        args:
          - "--max-line-length=120"
```